Anlab.Mvc/Client/components/TestList.tsx

```diff
@@ -70,13 +70,15 @@ export class TestList extends React.Component<ITestListProps, ITestListState> {
                 const selected = !!this.props.selectedTests[item.id];
                 const priceDisplay = (this.props.payment.clientType === 'uc' ? item.internalCost : item.externalCost);
                 const tooltipContent = converter.makeHtml(item.notes);
+                const url = '/analysis/' + item.category + '/' + item.sop;
```

Comment:
@jSylvestre more out of curiosity than anything else, does string interpolation work here in typescript?  So it would be 

            const url = `/analysis/${item.category}/${item.sop}`;

Reply:
@srkirkland Interesting. works with ` but not '

Reply:
@jSylvestre correct, backticks are required for string interpolation, otherwise they are just regular strings.  JavaScript history i supposed